deps.edn

```diff
@@ -1,8 +1,8 @@
 {:paths ["src"]
  :deps {org.clojure/clojure {:mvn/version "1.10.0"}
-        juxt/crux-core {:mvn/version "19.07-1.3.0-alpha"}
-        org.jetbrains.xodus/xodus-openAPI {:mvn/version "1.3.91"}
-        org.jetbrains.xodus/xodus-environment {:mvn/version "1.3.91"}}
+        juxt/crux-core {:mvn/version "20.09-1.11.0-beta"}
```

Comment:
Er is een nog nieuwere versie van crux beschikbaar

Reply:
Yes daar ga ik een losse PR voor maken